dashboard-parent/dashboard-core/pom.xml

```diff
@@ -16,6 +16,7 @@
 
 	<properties>
 		<xstream.version>1.4.10</xstream.version>
+		<jetty.version>9.2.22.v20170606</jetty.version>
```

Comment:
This is not required, defined in parent pom

Reply:
This was not working so far. But now i got it.